scripts/pre_commit_linter.py

```diff
@@ -447,8 +447,9 @@
     {
         'regexp': re.compile(r'\Wstr\('),
         'message': (
-            'Please try to use python_utils.UNICODE. If you are getting an '
-            'error, use python_utils.convert_to_bytes() instead.'),
+            'Please try to use python_utils.UNICODE or '
```

Comment:
Drop mention of python_utils.UNICODE here -- we use convert_to_bytes() right?

Reply:
So people should not use UNICODE at all? What about when converting an int to str?

Reply:
Ah ok, how about saying:

Please try to use python_utils.convert_to_bytes() for the strings used in webapp2's built-in methods or for strings used directly in NDB datastore models. If you need to cast ints/floats to strings, please use python_utils.UNICODE() instead.

Reply:
Done